projects/ad7124_iio/app/ad7124_iio.c

```diff
@@ -350,10 +349,10 @@ static char *ad7124_power_mode[] = {
 /************************** Functions Definition *****************************/
 /*****************************************************************************/
 /**
- * @brief Get the IIO scale for input channel
- * @param chn[in] - Input channel
- * @param scale[in,out] - Channel IIO scale value
- * @return 0 in case of success, negative error code otherwise
+ * @brief	Get the IIO scale for input channel
```

Comment:
What are these spacing changes about? Could we remove them if they aren't intentional?